examples/07-afqmc_usual_keywords.py

```diff
@@ -0,0 +1,38 @@
+import os
+import numpy as np
+
+from pyscf import gto, scf, cc
+from ad_afqmc import config
+config.afqmc_config["use_mpi"] = True # To run with MPI
```

Comment:
If mpi4py is available, it should be used by default. So it's not necessary to set this flag, right?